awscli/customizations/utils.py

```diff
@@ -66,11 +66,30 @@ def rename_command(command_table, existing_name, new_name):
     del command_table[existing_name]
 
 
-def alias_command(command_table, existing_name, new_name):
-    """Moves an argument to a new name, keeping the old as a hidden alias."""
+def alias_command(command_table, existing_name, new_name, hide_current=True):
+    """Moves an argument to a new name, keeping the old as a hidden alias.
+
+    :type command_table: dict
+    :param command_table: The full command table for the CLI or a service.
+
+    :type existing_name: str
+    :param existing_name: The current name of the command.
+
+    :type new_name: str
+    :param new_name: The new name for the command.
+
+    :type hide_current: bool
```

Comment:
We use `make_hidden_alias` for doing this same thing with parameters.  Shouldn't we just have a `make_hidden_command_alias` as well to be consistent?  I had to read the docs a few times to understand what `hide_current` meant.